### ISSUE TITLE
ERROR: relation "cohort_generation_info" does not exist   Position: 473

### ISSUE THREAD

**Issue by javier-gracia-tabuenca-tuni** [OHDSI/WebAPI#1362]
Followed the instructions to install WebAPI on a linux server with : 
- postgres
- pgAdmin4
- apache-maven-3.6.2
- apache-tomcat-8.5.47
- jdk1.8.0_231
- WebAPI-2.7.4

On the  deployment of webapi.war stops with the following error : 

```
Caused by: org.postgresql.util.PSQLException: ERROR: relation "cohort_generation_info" does not exist   Position: 473
```
On tomcat and postgres servers. 

When cheking the database I can see that all tables, inducing "cohort_generation_info" table have been created successfully.  

Following I attach the logs from tomcat 

[log2.txt](https://github.com/OHDSI/WebAPI/files/3863907/log2.txt)

Any idea what this can be,??

Thanks, Javier

**Comment by anthonysena** [comment]
Hi @javier-gracia-tabuenca-tuni - can you confirm the version of PostgreSQL that you are using? I was comparing your log file to one that I have from my own internal development environment. I noticed that your log file makes mention of PostgreSQL 13.0:

> 2019-11-19 14:54:47.637 INFO localhost-startStop-1 org.flywaydb.core.internal.util.VersionPrinter -  - Flyway 4.2.0 by Boxfuse
> 2019-11-19 14:54:47.760 INFO localhost-startStop-1 org.flywaydb.core.internal.dbsupport.DbSupportFactory -  - Database: jdbc:postgresql://localhost:5432/OHDSI (PostgreSQL 13.0)

My log file is similar except that this reference is PostgreSQL 10.1 which is what I'm currently using. 

I also wanted to confirm that you assigned the default permissions to the `ohdsi_app_user` as described on the wiki [here](https://github.com/OHDSI/WebAPI/wiki/PostgreSQL-Installation-Guide#prepare-database-schema-for-webapi). This may also be a source of your issue.

**Comment by javier-gracia-tabuenca-tuni** [comment]
Hey @anthonysena, thanks for the quick answer. 

Yes, I gitted Postgress from the github with the last version (actually, I just realized that forgot to set a version before compiling so it is 13.devs). 

Yes, I'm pretty sure I set the permissions right. I tried with two different data bases , once configured manually using pgAdmin4 and the second one directly copy pasting the SQL commands from the guide.  Both gave the same error. 

I 'm gonna try downgrading to PostgresSQL 10, if you think that that may be the problem. 

let u know soon
